containerz/containerz.proto

```diff
@@ -257,6 +343,50 @@ message StartContainerRequest {
   // List of volumes that should be attached to the container.
   repeated Volume volumes = 7;
 
+  // The network to attach this container to. This could be 'host', 'bridged',
+  // or any other network available in the runtime.
+  string network = 8;
+
+  message Capabilities {
```

Comment:
Can you document what capabilities are being described here? I didn't find any docs here or in the README.

(I assume that this covers capabilities(7), but I'm not sure -- if so, can't this be an enum?)